### ISSUE TITLE
Remove pervasive shift+enter key shortcut

### ISSUE THREAD

**Issue by jjant** [dramforever/vscode-ghc-simple#83]
I spent some time trying to find what was breaking my ability to use shift+enter to do stuff (move to previous result in search, for example). Turns out this extension hijacks this for the inline-repl thing (which doesn't even work for me).

Please remove this or at least add it in the settings.

**Comment by dramforever** [comment]
Oops, I didn't actually realize that implementing a shortcut key worked like that. Sorry about this. I'll see if I can say fall back to other shortcut assignments when there's nothing to do.

Until a fix is ready though, I suppose you can disable shift+enter within the keyboard shortcut settings or move it to a different key. It's pretty customizable after all.

**Comment by dramforever** [comment]
Speaking of inline-repl not working, do you have any specifics? Is this confusing/not running?

> Inline REPL: Add REPL blocks to your code with haddock syntax, either using no spaces before `>>>` or put it in a comment like `-- >>>`. Click on the code lens or type `Shift+Enter` to run a single block, or type `Shift+Alt+Enter` to run all blocks in a file.

You can open a new issue about that, or just tell me about it here.

**Comment by dramforever** [comment]
I think I found out what happened.  I wrote this as the condition for the shortcut key:

```
editorLangId == haskell && config.ghcSimple.feature.inlineRepl
```

however, I did not realize that this would still apply if the user is focused on the search bar. A `editorTextFocus` is needed for the intention of 'user is in editor'.

Also disabling the inline REPL feature in the extension settings should also turn it off. Unchecking this should work:

![Screenshot_20201023_132841](https://user-images.githubusercontent.com/2818072/96959926-b0e14200-1533-11eb-9ea5-a9864d13acfd.png)

Does this satisfy what you meant by 'at least add it in the settings'?
